media-gfx/opentoonz/opentoonz-1.2.0.recipe

```diff
@@ -71,31 +76,50 @@ BUILD_PREREQUIRES="
 	cmd:aclocal
 	cmd:autoconf
 	cmd:automake
+	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
 	cmd:libtoolize$secondaryArchSuffix
-	cmd:cmake
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage opentoonz$secondaryArchSuffix \
+	"$appsDir"/OpenToonz/lzocompress \
+	"$appsDir"/OpenToonz/lzodecompress \
+	"$appsDir"/OpenToonz/opentoonz_1.2 \
+	"$appsDir"/OpenToonz/tcleanup \
+	"$appsDir"/OpenToonz/tcomposer \
+	"$appsDir"/OpenToonz/tconverter \
+	"$appsDir"/OpenToonz/tfarmcontroller \
+	"$appsDir"/OpenToonz/tfarmserver
+
 BUILD()
 {
 	cd thirdparty/tiff-4.0.3
 	autoreconf -fi
 	./configure --with-pic --disable-jbig
 	make $jobArgs
+
 	cd ../../toonz
 	mkdir -p build && cd build
+	mkdir -p $appsDir/OpenToonz
+
 	cmake ../sources \
 		-DCMAKE_BUILD_TYPE=Release \
-        -DCMAKE_INSTALL_PREFIX=$prefix \
+        -DCMAKE_INSTALL_PREFIX=$appsDir/OpenToonz \
         -DGLUT_LIB=/system/develop/lib/libglut.so \
         -DCMAKE_CXX_FLAGS=-DBOOST_NO_CXX11_CONSTEXPR
```

Comment:
Can you please change the spaces into tabs?

Also, `portPackageLinksDir` should be used for cross-referencing between packages. And can I get an explaination for why we keep passing `-DBOOST_NO_CXX11_CONSTEXPR` to the C++ compiler? AFAIK, it's only required if the compiler is buggy regarding C++11 `constexpr` support.

```diff
-        -DCMAKE_INSTALL_PREFIX=$appsDir/OpenToonz \
-        -DGLUT_LIB=/system/develop/lib/libglut.so \
-        -DCMAKE_CXX_FLAGS=-DBOOST_NO_CXX11_CONSTEXPR
+		-DCMAKE_INSTALL_PREFIX=$appsDir/OpenToonz \
+		-DGLUT_LIB=$portPackageLinksDir/devel~libglut/develop/lib/libglut.so

Reply:
'Cause Haiku and/or GCC5.4.0 doesn't support it. Without passing this it will bail out with errors.
Afaik GCC6 solves this, but i'm not educated in this field. For sure it isn't copy-pasta, but tested and required flag, at least for now.

Reply:
Btw, "$portPackageLinksDir/devel~libglut/develop/lib/libglut.so" is this thing secondary_arch friendly? Seems to be pretty hardcoded for me.

Reply:
Thanks for the explaination.